app/kumactl/cmd/apply/apply.go

```diff
@@ -138,9 +139,15 @@ $ kumactl apply -f https://example.com/resource.yaml
 						return err
 					}
 				} else {
-					if err := upsert(cmd.Context(), pctx.Runtime.Registry, rs, resource); err != nil {
+					warnings, err := upsert(cmd.Context(), pctx.Runtime.Registry, rs, resource)
+					if err != nil {
 						return err
 					}
+					for _, w := range warnings {
+						if _, err := fmt.Fprintf(cmd.ErrOrStderr(), "Warning: %v\n", w); err != nil {
```

Comment:
Worth adding a `#` ?

Reply:
sorry not sure I understand, you mean adding `#` somewhere in the output line?

Reply:
That way if we return a yaml the warning is a comment. Though this is Stderr so it's fine